shared/fs/banner/widget-banner/index.js

```diff
@@ -0,0 +1,58 @@
+// @flow
+import * as React from 'react'
+import * as Kb from '../../../common-adapters'
+import * as Styles from '../../../styles'
+
+export type BannerType = 'warning' | 'failure' | 'none'
+type Props = {
+  onHide: () => void,
+  onRetry: () => void,
+  bannerType: BannerType,
+}
+
+const Banner = (props: Props) =>
+  props.bannerType !== 'none' && (
+    <Kb.Box2
+      direction="vertical"
+      fullWidth={true}
+      centerChildren={true}
+      style={props.bannerType === 'warning' ? styles.warning : styles.failure}
+    >
+      {props.bannerType === 'warning' ? (
+        <>
+          <Kb.Text type="BodySemibold" style={styles.text}>
+            You have less than 1 GB of storage space. Make some space, or unsync some folders.
+          </Kb.Text>
+          <Kb.Text type="BodySemibold" style={styles.text} underline={true} onClick={props.onHide}>
+            Hide this
+          </Kb.Text>
+        </>
+      ) : (
+        <Kb.Text type="BodySemibold" style={styles.text}>
+          You are out of storage space. Make some space then{' '}
+          <Kb.Text onClick={props.onRetry} underline={true} type="BodySemibold" style={styles.text}>
+            retry the sync
+          </Kb.Text>{' '}
+          or unsync some folders.
+        </Kb.Text>
+      )}
+    </Kb.Box2>
```

Comment:
Same here; can we use `Kb.Banner` here? Also let's move this to `menubar/` as that's where the `OutOfDate` banner is as well.